AF 22 - Alteração em conexão para conversão de caracteres

warleyxavier opened the pull request:
O erro ocorria devido a que o motor de execução não estava conseguindo converter os caracteres com acento para o padrão esperado pelo banco, fazendo-se necessário inserir uma configuração na configuração informando que o CharacterSet esperado é o WIN1252

Commit:
AF 22 - Alteração em conexão para converção de caracteres

O erro ocorria devido a que o motor de execução não estava conseguindo converter os caracteres com acento para o padrão esperado pelo banco, fazendo-se necessário inserir uma configuração na configuração informando que o CharacterSet esperado é o WIN1252

msnts commented on src/command/db/common/Eagle.Alfred.Command.DB.Common.FiredacConnection.pas:
```diff
@@ -76,6 +76,8 @@ procedure TFireDacFirebirdConnection.CreateConnection;
   FConnection.Params.Add('Password=' + FPassword);
   FConnection.Params.Add('Port=' + FPort);
 
+  FConnection.Params.Add('CharacterSet=WIN1252');
```
Isso precisa ser configurável

Commit:
AF 22 - Implementação opção de alteração do characterSet

Implementada possibilidade de alteração do characterset de acordo com a necessidade do desenvolvedor.
